lib/puppet/resource/catalog.rb

```diff
@@ -4,6 +4,8 @@
 require 'puppet/util/tagging'
 require 'puppet/graph'
 
+require_relative 'capability_finder'
```

Comment:
I thought that was best practice after Ruby 1.9, as it prevents various loadp ath shenanigans; changed it to an absolute require